### ISSUE TITLE
Different values for same item?

### ISSUE THREAD

**Issue by tanql** [coreylynch/pyFM#19]
After training the model, I get different values of the same item, when I enter more items to the predicting list? Is this supposed to happend or is it a bug?
For eksample:
print fm.predict(v.fit_transform({"user": "1", "item": "10", "age": 24}))

print fm.predict(v.fit_transform([{"user": "1", "item": "10", "age": 24},{"user": "1", "item": "12", "age": 24}]))

both have user 1 and item 10, however the ratings of those would be different...

**Comment by sunxiang0301** [comment]
It is because in line 179 in pylibfm.py, the 'random_state' param  in  cross_validation.train_test_split is not specified. Modify it to 
cross_validation.train_test_split(
            X, y, test_size=self.validation_size, random_state=self.seed)
reinstall, and the problem fixed.
